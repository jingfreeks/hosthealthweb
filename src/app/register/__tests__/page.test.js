import { render } from "@testing-library/react";
import Page from "../page";

describe("Register Screen Page", () => {
  it("renders Register Screen Page to match snaptshots", async() => {
    const { container } = render(<Page />);
    expect(container).toMatchSnapshot();
  });
});
