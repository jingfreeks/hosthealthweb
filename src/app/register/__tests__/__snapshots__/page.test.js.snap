// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Register Screen Page renders Register Screen Page to match snaptshots 1`] = `
<div>
  <div
    class="h-screen w-screen flex justify-center items-center bg-slate-100"
  >
    <div
      class="shadow-xl px-8 pb-8 pt-12 bg-white rounded-xl space-y-12"
    >
      <h1
        class="font-semibold text-2xl"
      >
        Create your Account
      </h1>
      <form
        class="space-y-12 w[400px]"
      >
        <div
          class="grid w-full max-w-sm items-center gap-1.5"
        >
          <label
            class="text-sm font-medium leading-none peer-disabled:cursor-not-allowed peer-disabled:opacity-70"
            for="email"
          >
            Email
          </label>
          <input
            class="flex h-10 w-full rounded-md border border-input bg-background px-3 py-2 text-sm ring-offset-background file:border-0 file:bg-transparent file:text-sm file:font-medium placeholder:text-muted-foreground focus-visible:outline-none focus-visible:ring-2 focus-visible:ring-ring focus-visible:ring-offset-2 disabled:cursor-not-allowed disabled:opacity-50"
            id="email"
            type="email"
            value=""
          />
        </div>
        <div
          class="grid w-full max-w-sm items-center gap-1.5"
        >
          <label
            class="text-sm font-medium leading-none peer-disabled:cursor-not-allowed peer-disabled:opacity-70"
            for="password"
          >
            Password
          </label>
          <input
            class="flex h-10 w-full rounded-md border border-input bg-background px-3 py-2 text-sm ring-offset-background file:border-0 file:bg-transparent file:text-sm file:font-medium placeholder:text-muted-foreground focus-visible:outline-none focus-visible:ring-2 focus-visible:ring-ring focus-visible:ring-offset-2 disabled:cursor-not-allowed disabled:opacity-50"
            id="password"
            type="password"
            value=""
          />
        </div>
        <div
          class="grid w-full max-w-sm items-center gap-1.5"
        >
          <label
            class="text-sm font-medium leading-none peer-disabled:cursor-not-allowed peer-disabled:opacity-70"
            for="password"
          >
            Confirm Password
          </label>
          <input
            class="flex h-10 w-full rounded-md border border-input bg-background px-3 py-2 text-sm ring-offset-background file:border-0 file:bg-transparent file:text-sm file:font-medium placeholder:text-muted-foreground focus-visible:outline-none focus-visible:ring-2 focus-visible:ring-ring focus-visible:ring-offset-2 disabled:cursor-not-allowed disabled:opacity-50"
            id="password"
            type="password"
            value=""
          />
        </div>
        <div
          class="w-full"
        >
          <button
            class="inline-flex items-center justify-center whitespace-nowrap text-sm font-medium ring-offset-background transition-colors focus-visible:outline-none focus-visible:ring-2 focus-visible:ring-ring focus-visible:ring-offset-2 disabled:pointer-events-none disabled:opacity-50 bg-primary text-primary-foreground hover:bg-primary/90 h-11 rounded-md px-8 w-full"
          >
            Register
          </button>
        </div>
      </form>
      <p
        class="text-center"
      >
        Have an account? 
        <a
          class="text-indigo-500 hover:underline"
          href="/login"
        >
          Sign in
        </a>
      </p>
    </div>
  </div>
</div>
`;
